data/json/items/comestibles/sandwich.json

```diff
@@ -110,6 +110,24 @@
     "fun": 15,
     "vitamins": [ [ "vitC", 6 ], [ "calcium", 7 ], [ "iron", 12 ] ]
   },
+  {
+    "type": "COMESTIBLE",
+    "id": "sandwich_fairy",
+    "name": "fairy bread",
+    "name_plural": "fairy bread",
+    "weight": 114,
+    "color": "brown",
+    "spoils_in": "3 days",
+    "container": "wrapper",
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "description": "An open-faced sandwich consisting of sliced white bread, a healthy crust-to-crust slathering of butter, and sprinkles.  Supposedly a staple of birthday parties in Australia.",
+    "price": 85,
+    "material": [ "milk", "wheat" ],
```

Comment:
I'm going to go out on a limb here and suggest that fairy bread may even count as 'junk food'.

Reply:
Done. It's now junk food.